Description: Change version
 We imported a snapshot to get latest improvements. Change the dmks.conf
 version to be consistent with source package version.
Author: Sophie Brun <sophie@offensive-security.com>
Last-Update: 2019-08-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/dkms.conf
+++ b/dkms.conf
@@ -1,5 +1,5 @@
 PACKAGE_NAME="realtek-rtl88xxau"
-PACKAGE_VERSION="5.2.20.2~20190429"
+PACKAGE_VERSION="5.2.20.2~20190617"
 CLEAN="'make' clean"
 BUILT_MODULE_NAME[0]=88XXau
 PROCS_NUM=`nproc`
